emperor/util.py

```diff
@@ -99,15 +95,17 @@ def copy_support_files(file_path):
     # directory into another existing directory, hence the system call.
     # use double quotes for the paths to escape any invalid chracter(s)/spaces
     cmd = 'cp -R "%s/"* "%s"' % (get_emperor_support_files_dir(),
-        abspath(file_path))
+                                 abspath(file_path))
     cmd_o, cmd_e, cmd_r = qcli_system_call(cmd)
 
     if cmd_e:
-        raise EmperorSupportFilesError, "Error found whilst trying to copy " +\
-            "the support files:\n%s\n Could not execute: %s" % (cmd_e, cmd)
+        raise EmperorSupportFilesError(("Error found whilst trying to copy ",
+                                       " the support files:\n{}".format(cmd_e),
```

Comment:
Can you remove this comma? The exception message being passed is a tuple of two strings instead of a single string. On that same note can you remove the parenthesis surrounding the message?